lib/src/components/_stacks-tables.less

```diff
@@ -197,6 +254,34 @@
     }
 }
 
+//  $$  Simple THEAD
+//  ------------------------------------------------------------------------
+.s-table__simple-thead {
+    thead {
+        th {
+            background-color: transparent;
+            border-top: none;
+            border-right: none;
+        }
+    }
+}
+
+//  $$  Sortable Table Columns
+//  ------------------------------------------------------------------------
+.s-table__sortable {
```

Comment:
I would make this a table-level class. The state `.is-sorted` would apply to the `th` still.